tests/integration/conftest.py

```diff
@@ -116,6 +116,12 @@ def eth2_up():
         yield ifstate
 
 
+@pytest.fixture(scope="function")
+def lldptest_up():
```

Comment:
Can you use fixture to crate veth pair for LLDP test in the run time like DHCP test did?

Reply:
Done, LLDP tests is creating a veth pair in the runtime. Thanks!